src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -423,6 +430,17 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
           this.sessionEditFormModel.timeZone, true);
     }
 
+    this.deleteDeadlineExtensionsHandler(submissionEndTime).subscribe((isUpdateSession) => {
+      if (isUpdateSession) {
+        this.updateFeedbackSession(submissionStartTime, submissionEndTime, sessionVisibleTime, responseVisibleTime);
+      }
+    });
```

Comment:
It's possible, but I thought to leave the `updateFeedbackSession` in the higher-level `editExistingSessionHandler` instead of `deleteDeadlineExtensionsHandler`. It might be strange to pass all the arguments (`sessionVisibleTime` etc) to `deleteDeadlineExtensionsHandler` too

Reply:
Actually, was thinking of just removing the if statement but leaving `updateFeedbackSession` here, but I think nevermind, this is probably okay as it is already.